lib/hydra/works/virus_scanner.rb

```diff
@@ -0,0 +1,32 @@
+# Provides a mechanism for defining a virus scanner.
+# If ClamAV is present, we'll use that; otherwise, we use ::null_scanner. If you want to use a
+# virus checker other than ClamAV, you'll need to configure it by creating your own class
+# that responds to ::call and setting in Hydra::Works. For example:
+#
+#   class MyScanner
+#     def self.call(file)
+#       Scanner.check_for_viruses(file)
+#     end
+#   end
+#
+#   Hydra::Works.default_system_virus_scanner = MyScanner
```

Comment:
Do we want to recommend that folks set this in an initializer in Rails environments? Or meh? I could go either way on that.